QMLComponents/boundcontrols/boundcontrolmultiterms.cpp

```diff
@@ -71,3 +73,36 @@ void BoundControlMultiTerms::resetBoundValue()
 
 	setBoundValue(boundValue);
 }
+
+void BoundControlMultiTerms::setBoundValue(const Json::Value &value, bool emitChanges)
+{
+	Json::Value newValue;
+
+	if (_control->encodeValue())
+	{
+		if (_isValueWithTypes(value))
+			newValue = value;
+		else
+		{
+			Json::Value types(Json::arrayValue);
```

Comment:
So this "else" is *just* for loading pre-0.19.2 versions no?

Reply:
If so maybe add a comment here a la 
```suggestion
			// Else we are loading from a jasp version before "preloadData" was on or var.types were added to the options
			Json::Value types(Json::arrayValue);
```